### PR TITLE
Fix libpmem detection in case of pkg_config is missing

### PR DESCRIPTION
**Pull request by michalbiesek**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pelikan/14)

### CODE REVIEW

**Review by pbalcer — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -138,7 +138,7 @@ if (USE_PMEM)
     if(PKG_CONFIG_FOUND)
         pkg_check_modules(LIBPMEM REQUIRED libpmem>=1.0)
     else()
-        find_package(LIBPMEM REQUIRED 1.0)
+        find_package(LIBPMEM QUIET REQUIRED 1.0)
```
Comment (pbalcer) [inline CMakeLists.txt]: why did you add quiet? this should fail.

Comment (michalbiesek) [reply CMakeLists.txt]: Done ( I removed QUIET)
